src/pages/entry.js

```diff
@@ -1,19 +1,53 @@
-import React from 'react'
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import React, { Fragment } from 'react'
 import { withRouter } from 'react-router-dom'
+import styles from './entry.styles'
 import Anchor from 'components/Anchor'
+import Box from 'components/Box'
+import LargeHeader from 'components/Header/LargeHeader'
+import ListLink from 'components/ListLink'
+import Text from 'components/Text'
+import { TEXT_TYPE } from 'components/Text/constants'
 
 class EntryPortal extends React.Component {
   constructor(props) {
-    super(props);
+    super(props)
   }
 
   render() {
     return (
-      <div>
-        <Anchor href="/request">Need</Anchor>
-      </div>
+      <Fragment>
+        <LargeHeader />
+        <Box css={styles.box}>
+          <ListLink
+            href={'/request'}
+            title={'Need'}
+            text={'Healthcare professionals only'}
+          />
+          <ListLink
+            href={'/donate'}
+            title={'Donate'}
+            text={'Open to the public'}
+          />
+          <div css={styles.titleAndText}>
+            <Text css={styles.title} as={'h3'} type={TEXT_TYPE.HEADER_4}>
+              Nurses and doctors need supplies to keep fighting the coronavirus.
+              We help them communicate their needs to people who can help.
+            </Text>
+            <Text css={styles.text} as={'p'} type={TEXT_TYPE.BODY_2}>
+              Our platform gives a real-time view of immediate needs at various
+              hospitals. We make the information freely available so anyone can
+              help, including organizations who can provide supplies at scale.
+            </Text>
+          </div>
+          <Anchor css={styles.anchor} href="/request" withIcon>
```

Comment:
Do we agree on using a list of routes in a constant file?

```suggestion
          <Anchor css={styles.anchor} href={ROUTES.REQUEST} withIcon>
```

Reply:
I'm plus +100 for this and am happy to take the task to convert everything over.

Reply:
Great suggestion, thanks!

Reply:
@joshrounsville I added the Routes file, and the three routes in this page, but if you have time maybe you could do a sweep of the rest of the app after I merge this to see if there are more?

Reply:
Will do!